bears/general/InvalidLinkBear.py

```diff
@@ -10,22 +10,23 @@
 class InvalidLinkBear(LocalBear):
 
     @classmethod
-    def check_prerequisites(cls):
-        code = InvalidLinkBear.get_status_code_or_error("http://www.google.com")
+    def check_prerequisites(cls, timeout=2):
```

Comment:
there's no way a user can change the default value, so I would just leave the param here^^